keps/693-multikueue/README.md

```diff
@@ -316,6 +323,76 @@ that ware created by this multikueue manager cluster and delete them if their lo
 - `WorkerLostTimeout` - defines the time a local workload's multikueue admission check state is kept Ready
 if the connection with its reserving worker cluster is lost.
 
+
+### MultiKueue Dispatcher API
+
+The MultiKueue Workload Controller consists of nomination nd synchronization: 
+first, it nominates worker clusters for each workload based on the configured dispatching mode;
+then, it synchronizes the workload by creating and managing its copies in the nominated clusters. 
+This ensures that only one cluster ultimately admits and runs the workload, 
+while the management cluster maintains up-to-date status by tracking the admitted job.
+
+#### Cluster Nomination
+Cluster nomination phase looks differently depending on the configured dispatching mode.
+MultiKueue supports three dispatching modes: AllAtOnce (`kueue.x-k8s.io/multikueue-dispatcher-all-at-once`),
+Incremental (`kueue.x-k8s.io/multikueue-dispatcher-incremental`), and External.
+
+* **AllAtOnce Mode**:  
+  In this mode, all available clusters are considered for dispatching at once, and the workload is copied to each worker cluster.
+
+* **Incremental Mode**:  
```

Comment:
How to manage if clusters have already been nominated in the previous round?
How to sort the available clusters? alphabetical order?